bokeh/document.py

```diff
@@ -431,3 +469,30 @@ def validate(self):
             refs = r.references()
             root_sets.append(refs)
             check_integrity(refs)
+
+    @property
+    def session_callbacks(self):
+        return self._session_callbacks.values()
```

Comment:
:) yes, I was thinking about duplicating the events but you are right.. not real need to do it. Better have a higher level `SessionCallback*` event.